statistics/handle/cache/internal/lru/lru_cache.go

```diff
@@ -1,443 +0,0 @@
-// Copyright 2023 PingCAP, Inc.
```

Comment:
Better to keep it unchanged, we can remove it in the next few PRs after finishing the new LFU completely.

Reply:
Done, it has been recovered.